src/components/accounts/Login.js

```diff
@@ -34,8 +33,12 @@ export class Login extends Component {
                 <div className="card card-body mt-5">
                     <h2 className="text-center">Login</h2>
                     <form onSubmit={this.onSubmit}>
+                        {this.props.errors.msg.non_field_errors ?
+                            <span className='alert alert-danger'>{this.props.errors.msg.non_field_errors}</span>
+                            : null}
+
                         <div className="form-group">
-                            <label>Username</label>
+                            <label className="mt-2">Username*</label>
```

Comment:
Here are accessible ways to make a field required: https://dequeuniversity.com/tips/identify-required-elements
Use any of these.

Helpful link: https://www.w3.org/TR/WCAG20-TECHS/ARIA2.html

Reply:
I've added this:
<p id="formInstructions">Fields marked with an asterisk (*) are required.</p>